src/overlays/actors/ovl_En_Ice_Hono/z_en_ice_hono.c

```diff
@@ -21,37 +32,345 @@ const ActorInit En_Ice_Hono_InitVars = {
+static ColliderCylinderInit sCylinderInitCapturableFlame = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 25, 80, 0, { 0, 0, 0 } },
+};
+
+static ColliderCylinderInit sCylinderInitDroppedFlame = {
+    { COLTYPE_UNK10, 0x21, 0x00, 0x21, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0xFFCFFFFF, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x01, 0x00, 0x01 },
+    { 12, 60, 0, { 0, 0, 0 } },
+};
+
+static InitChainEntry sInitChainCapturableFlame[] = {
+    ICHAIN_U8(unk_1F, 0, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_4C, 60, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 400, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+static InitChainEntry sInitChainDroppedFlame[] = {
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 400, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+static InitChainEntry sInitChainSmallFlame[] = {
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 400, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+f32 EnIceHono_SquareDist(Vec3f* pos1, Vec3f* pos2) {
+    f32 dx = pos1->x - pos2->x;
+    f32 dz = pos1->z - pos2->z;
+    
+    return SQ(dx) + SQ(dz);
+}
+
+void EnIceHono_InitCapturableFlame(Actor* thisx, GlobalContext* globalCtx) {
+    EnIceHono* this = THIS;
+
+    Actor_ProcessInitChain(&this->actor, sInitChainCapturableFlame);
+    Actor_SetScale(&this->actor, 0.0074f);
+    this->actor.flags |= 1;
+    Actor_SetHeight(&this->actor, 10.0f);
+
+    Collider_InitCylinder(globalCtx, &this->collider);
+    Collider_SetCylinder(globalCtx, &this->collider, &this->actor, &sCylinderInitCapturableFlame);
+    Collider_CylinderUpdate(&this->actor, &this->collider);
+
+    this->actor.colChkInfo.mass = 0xFF;
+    EnIceHono_SetupActionCapturableFlame(this);
+}
+
+void EnIceHono_InitDroppedFlame(Actor* thisx, GlobalContext* globalCtx) {
+    EnIceHono* this = THIS;
+
+    Actor_ProcessInitChain(&this->actor, sInitChainDroppedFlame);
+    this->actor.scale.x = this->actor.scale.z = this->actor.scale.y = 0.00002f;
+    this->actor.gravity = -0.3f;
+    this->actor.minVelocityY = -4.0f;
+    this->actor.shape.unk_08 = 0.0f;
+    this->actor.shape.rot.x = this->actor.shape.rot.y = this->actor.shape.rot.z = this->actor.posRot.rot.x =
+        this->actor.posRot.rot.y = this->actor.posRot.rot.z = 0;
+
+    Collider_InitCylinder(globalCtx, &this->collider);
+    Collider_SetCylinder(globalCtx, &this->collider, &this->actor, &sCylinderInitDroppedFlame);
+    Collider_CylinderUpdate(&this->actor, &this->collider);
+
+    this->collider.dim.radius = this->actor.scale.x * 4000.4f;
+    this->collider.dim.height = this->actor.scale.y * 8000.2f;
+    this->actor.colChkInfo.mass = 0xFD;
```

Comment:
cf `func_8006146C` the only special mass values are 0xFE and 0xFF so this should be decimal? (`0xFD = 253`)
but it's also special because it's the maximum non-special mass so I don't know lol